account_product_fiscal_classification/tests/tests.py

```diff
@@ -138,3 +144,12 @@ def test_06_unlink_fiscal_classification(self):
             [self.classification_1_id])
         with self.assertRaises(ValidationError):
             classification.unlink()
+
+    def test_07_classification_generate_from_template(self):
```

Comment:
@danimaribeiro good catch.
Now does anyone understands why this test doesn't run? I'm not getting it...